banzai_nres/dbs.py

```diff
@@ -163,3 +191,19 @@ def get_resource_file(db_address, key):
     with banzai.dbs.get_session(db_address=db_address) as db_session:
         resource_file = db_session.query(ResourceFile).filter(ResourceFile.key == key).first()
     return resource_file
+
+
+def get_closest_existing_classification(db_address, ra, dec):
+    with banzai.dbs.get_session(db_address=db_address) as db_session:
+        order = [Classification.diff_ra(ra), Classification.diff_dec(dec)]
+        model = db_session.query(Classification).order_by(*order).first()
```

Comment:
I might be misunderstanding the code, but don't we want the closest Euclidean distance in this case? Since RA and Dec have basically the same units, unlike for the stellar classification where we were dealing with quantities with very different units (Teff, etc).

Reply:
Added a proper great circle distance function.